### ISSUE TITLE
Variable which is defined in properties file is not accessible within StepDef which uses data table

### ISSUE THREAD

**Issue by sse2ebdd** [gwen-interpreter/gwen-web#91]
**Steps to reproduce -**
1. Create feature file with below content

```
Feature: Test

@DataTable(header="top")
@StepDef
Scenario: I test accessing variable in DSL which uses data table
And database name2 is "${dbSid}"

Scenario: Test
And database name is "${dbSid}"
And I test accessing varible in DSL which uses data table
| dummy_header |
| dummy_value  |
```

2. Create test.properties file with one property --> `dbSid=TEST`
3. Now run feature file (import properties using -c/-p option)
4. GWEN 3.0.8 is not able to access variable defined in properties file within StepDef which uses data table

![datatable-dsl-issue html](https://user-images.githubusercontent.com/86771461/144871780-1c88e827-d584-4933-8ae1-7adcbe813a1d.png)

**Env**
Gwen Version - 3.0.8
Chrome Driver - Latest (96)

**Comment by bjuric** [comment]
Fixed in v[3.0.9](https://github.com/gwen-interpreter/gwen-web/releases/tag/v3.0.9). Thanks

**Comment by sse2ebdd** [comment]
Verified this issue on GWEN v3.0.9.
It's working as expected, hence closing this issue.
